code/world.dm

```diff
@@ -23,14 +23,9 @@ var/global/datum/global_init/init = new ()
 var/global/list/map_transition_config = MAP_TRANSITION_CONFIG
 
 /world/New()
-	//logs
-	var/date_string = time2text(world.realtime, "YYYY/MM-Month/DD-Day")
-	href_logfile = file("data/logs/[date_string] hrefs.htm")
-	diary = file("data/logs/[date_string].log")
-	diaryofmeanpeople = file("data/logs/[date_string] Attack.log")
+	setLog()

```

Comment:
While you're at it, this should actually be moved to Line 10, so that logs are initialized before the world. There's a few things that write to the diary before the world, including the config controller.